docs/controls/AdaptiveGridView.md

```diff
@@ -10,23 +9,100 @@ keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, AdaptiveGridView,
+
+* Using `OneRowModeEnabled`, you can set `DesiredWidth` and `ItemHeight`, and the **width** of each item, and the number of visible columns will be adjusted on demand, using all horizontal space.
 
 ## Syntax
 
 ```xaml
-
 <controls:AdaptiveGridView  Name="AdaptiveGridViewControl"
           ItemHeight="200"
           DesiredWidth="300"
           ItemTemplate="{StaticResource PhotosTemplate}">
 </controls:AdaptiveGridView>
-
 ```
 
-## Example Image
+## Examples
+
+- Using `DesiredWidth` in combination with `ItemHeight`:
+
+    ![AdaptiveGridView DesiredHeight and ItemHeight](../resources/images/Controls/AdaptiveGridView/AdaptiveGridView-DesiredWidth&ItemHeight.gif "AdaptiveGridView DesiredHeight and ItemHeight")
+
+- Maintain aspect ratio by setting `DesiredWidth` with no `ItemHeight` set:
+
+    ![AdaptiveGridView Viewbox scaled](../resources/images/Controls/AdaptiveGridView/AdaptiveGridView-ViewboxAspectRatio.gif "AdaptiveGridView Viewbox scaled")
+
+    This still requires the `ItemTemplate` to contain some scaling logic, this can be done with Height and Width set on the content inside of a Viewbox, or using custom view logic.
+
+    - Using a `Viewbox`:
+
+        Using the `Height` and `Width` properties of content inside of a `Viewbox` means that the content inside will scale when **Width** and **Height** changes. This might not be a desired effect, and it will also likely incur a slight performance penalty.
+
+        _ItemTemplate implementation_
+
+        ```xaml
+        <DataTemplate x:Key="PhotosTemplate">
+            <Viewbox>
+                <Grid
+                    Height="300"
+                    Width="200"
+                    Background="White"
+                    BorderBrush="Black"
+                    BorderThickness="1">
+                    <Image
+                        Source="{Binding Thumbnail}"
+                        Stretch="UniformToFill"
+                        HorizontalAlignment="Center"
+                        VerticalAlignment="Center"/>
+                </Grid>
+            </Viewbox>
+        </DataTemplate>
+        ```
+
+    - To use custom view logic:
+
+        Using `MeasureOverride` on a ContentControl allows you to specify the **Width** and **Height** of the content, which might reap better performance compared to a `Viewbox`. The content inside will not scale when **Width** and **Height** changes.
+
+        _Custom logic implementation_
+
+        ```csharp
+        public class AspectContentControl : ContentControl
+        {
+            protected override Size MeasureOverride(Size availableSize)
+            {
+                return new Size(availableSize.Width, availableSize.Width * 1.6);
+            }
+        }
+        ```
+
+        _ItemTemplate implementation_
+
+        ```xaml
+        <DataTemplate x:Key="PhotosTemplate">
+            <controls:AspectContentControl 
```

Comment:
If this isn't something in the toolkit it should be a `local:` prefix to signify this is custom code they have in their own project.

Would we not want to add something like this and have a dependency property for the aspect ratio?

Reply:
@skendrot said that this was too insignificant to create, I agree with him.

Reply:
True, maybe when we get multi sample, we can just have a copy of it in the sample app code for juxtaposition and reference.